config_quick_debug.yml

```diff
@@ -20,3 +20,21 @@ scenario_params_list:
     - 5
    is_quick_demo:
     - True
+ - dataset_name:
+    - 'imdb'
+   partners_count:
+    - 3
+   amounts_per_partner: 
+    - [0.2, 0.5, 0.3]
+   samples_split_option:
+    - ['basic', 'random']
+   multi_partner_learning_approach:
+    - 'fedavg'
+   aggregation_weighting:
+    - 'uniform'
+   methods:
+    - ["Shapley values", "Independent scores"]
+   gradient_updates_per_pass_count:
+    - 5
+   is_quick_demo:
+    - True
```

Comment:
I wouldn't put it in the debug config. It has to stay a debug config. I think we should also remove CIFAR10 from there